Gruvo/Gruvo/ClientApp/src/app/profile/profile.service.ts

```diff
@@ -23,10 +24,14 @@ export class ProfileService {
   getUserTweets(): Observable<ITweet[]> {
     return this.http.get<ITweet[]>(this.profileTweetsApiURL);
   }
-  getSubscriptions(): Observable<IUser[]> {
-    return this.http.get<IUser[]>(this.profileSubscriptionsApiURL);
+  getSubscriptionsQuality(): Observable<number> {
```

Comment:
quality or quantity?) Better use word "count" for clarity) getSubscriptionsCount etc.

Reply:
accidentally)